tools/validators/instance_validator/tests/fake_instances/GOOD/update_links.yaml

```diff
@@ -32,7 +32,7 @@ VIRTUAL-ENTITY-GUID:
 PHYSICAL-ENTITY-GUID:
   type: HVAC/CHWS_WDT
   code: PHYSICAL-ENTITY
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.